cygnus-common/src/main/java/com/telefonica/iot/cygnus/backends/mysql/MySQLBackendImpl.java

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright 2015 Telefonica Investigación y Desarrollo, S.A.U
+ * Copyright 2016 Telefonica Investigación y Desarrollo, S.A.U
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1372/commits/87b02f0c7866879463e739ae58e02b54302386d2